config/0chain_blobber.yaml

```diff
@@ -69,6 +69,7 @@ block_worker: http://198.18.0.98:9091
 
 handlers:
   rate_limit: 0 # 10 per second . it can't too small one if a large file is download with blocks
+  file_rate_limit: 1000 # 100 files per second
```

Comment:
1000 is too fast for both upload/download by default, the blobber would be under DDoS, so perhaps 100. And if the upload/download process get affected, then we might need to optimize the block size to reduce the requests number.

Reply:
Make sense. Adjust the chunk size instead of creating too many API requests for upload, and block size for download.